test/functional/tests/example/example_test_power_cycle.py

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_plugin("power_control")
+def test_create_example_partitions():
+    """
+        title: Example test doing power cycle
+        description: |
+          Example usage of power_control plugin.
+          NOTE:
+          This test uses plugin that is not included in test-framework.
+          It should be provided by user as external_plugin.
+        pass_criteria:
+          - Should pass.
+    """
+    with TestRun.step("Power cycle DUT"):
+        print(TestRun.plugins)
+        power_control = TestRun.plugin_manager.get_plugin('power_control')
+        power_control.power_cycle()
```

Comment:
maybe add try-except block with TestRun.fail in except, so it wouldn't be exception as a test result, but fail

Reply:
Exception thrown by `power_cycle()` method will also fail the test. Plus it will produce more meaningful message.